<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="vl_modul">
        <item>
         <widget class="QGroupBox" name="gb_modul">
          <property name="title">
           <string>Modul</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QComboBox" name="cb_modul">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string>Wähle</string>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="placeholderText">
              <string>Wähle</string>
             </property>
             <item>
              <property name="text">
               <string>Modul 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 4</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_versuchart">
        <item>
         <widget class="QGroupBox" name="gb_versuchart">
          <property name="title">
           <string>Versuchsart</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <widget class="QComboBox" name="cb_versuchart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string>Wähle</string>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="placeholderText">
              <string>Wähle</string>
             </property>
             <item>
              <property name="text">
               <string>CU 17</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>UU 19</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_ablesungart">
        <item>
         <widget class="QGroupBox" name="bg_ablesungart">
          <property name="title">
           <string>Ablesungsart</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QComboBox" name="cb_ablesungart">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string>Wähle</string>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="placeholderText">
              <string>Wähle</string>
             </property>
             <item>
              <property name="text">
               <string>Manuelle Auswahl von Datenspalten</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Spaltennummern</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Namenerkennung</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_asci_lesen">
        <item>
         <widget class="QGroupBox" name="gb_asci_lesen">
          <property name="title">
           <string>ASCI Datei</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QPushButton" name="btn_asc_lesen">
             <property name="text">
              <string>Lesen</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vl_Ablesevorgang">
        <item>
         <widget class="QGroupBox" name="gb_Ablesevorgang">
          <property name="title">
           <string>Infos zum Ablesevorgang</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Datenspalten ausgelesen: X</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Datenzeilen ausgelesen: Y</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>50</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>21</number>
        </property>
        <property name="rightMargin">
         <number>44</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Angaben zum Versuch #1</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="placeholderText">
             <string>Probenhöhe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="placeholderText">
             <string>Probendurchmesser</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>Dichte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_20">
            <property name="font">
             <font>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;
&lt;span style=&quot; vertical-align:normal;&quot;&gt;kg/m&lt;/span&gt;
&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;
&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>-4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_9">
          <property name="placeholderText">
           <string>Material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_8">
          <property name="placeholderText">
           <string>Probenbezeichnung</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_7">
          <property name="placeholderText">
           <string>Testnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_6">
          <property name="placeholderText">
           <string>Projectnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_5">
          <property name="placeholderText">
           <string>Prüfdatum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_modul_2">
          <property name="title">
           <string>Vorlage aufrufen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QComboBox" name="cb_modul_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string>Wähle</string>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="placeholderText">
              <string>Wähle</string>
             </property>
             <item>
              <property name="text">
               <string>Modul 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 4</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Als Vorlage speichern</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <property name="leftMargin">
         <number>21</number>
        </property>
        <property name="rightMargin">
         <number>44</number>
        </property>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Angaben zum Versuch #1</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_34">
            <property name="placeholderText">
             <string>Probenhöhe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_35">
            <property name="placeholderText">
             <string>Probendurchmesser</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_36">
            <property name="placeholderText">
             <string>Dichte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_24">
            <property name="font">
             <font>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;
&lt;span style=&quot; vertical-align:normal;&quot;&gt;kg/m&lt;/span&gt;
&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;
&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>-4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_37">
          <property name="placeholderText">
           <string>Material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_38">
          <property name="placeholderText">
           <string>Probenbezeichnung</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_39">
          <property name="placeholderText">
           <string>Testnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_40">
          <property name="placeholderText">
           <string>Projectnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_41">
          <property name="placeholderText">
           <string>Prüfdatum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_modul_3">
          <property name="title">
           <string>Vorlage aufrufen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <widget class="QComboBox" name="cb_modul_3">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string>Wähle</string>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="placeholderText">
              <string>Wähle</string>
             </property>
             <item>
              <property name="text">
               <string>Modul 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 4</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Als Vorlage speichern</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_22">
        <property name="leftMargin">
         <number>21</number>
        </property>
        <property name="rightMargin">
         <number>44</number>
        </property>
        <item>
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>Angaben zum Versuch #1</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_34">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_74">
            <property name="placeholderText">
             <string>Probenhöhe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_35">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_75">
            <property name="placeholderText">
             <string>Probendurchmesser</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_43">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_36">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEdit_76">
            <property name="placeholderText">
             <string>Dichte</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_44">
            <property name="font">
             <font>
              <underline>false</underline>
              <strikeout>false</strikeout>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;
&lt;span style=&quot; vertical-align:normal;&quot;&gt;kg/m&lt;/span&gt;
&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;
&lt;/p&gt;
&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>-4</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_77">
          <property name="placeholderText">
           <string>Material</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_78">
          <property name="placeholderText">
           <string>Probenbezeichnung</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_79">
          <property name="placeholderText">
           <string>Testnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_80">
          <property name="placeholderText">
           <string>Projectnummer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_81">
          <property name="placeholderText">
           <string>Prüfdatum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="gb_modul_8">
          <property name="title">
           <string>Vorlage aufrufen</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_23">
           <item>
            <widget class="QComboBox" name="cb_modul_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string>Wähle</string>
             </property>
             <property name="currentIndex">
              <number>-1</number>
             </property>
             <property name="placeholderText">
              <string>Wähle</string>
             </property>
             <item>
              <property name="text">
               <string>Modul 1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Modul 4</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>Als Vorlage speichern</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
